src/components/BlogFeed.astro

```diff
@@ -67,6 +70,7 @@ type ArticleResult = {
             <img src={image.src} width={image.width} alt={media.data.alt} />
           </figure>
           <h4>{post.webTitle}</h4>
+          <h5><em>{post.fields.byline}</em></h5>
```

Comment:
I think it's worth italicising this via CSS rather than the `em` element given its purpose is cosmetic. The `em` element [is for emphasising important words or passages](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/em).